src/octoprint/plugins/eventmanager/templates/eventmanager_settings.jinja2

```diff
@@ -32,6 +32,13 @@
                 <select class="input-block-level" data-bind="value: event, options: $root.settingsViewModel.settings.plugins.eventmanager.availableEvents().sort(), optionsCaption: 'Select Event', valueAllowUnset: true"></select>
             </div>
         </div>
+
+        <div class="control-group">
+            <label class="control-label">{{ _('Custom Name') }}</label>
```

Comment:
It doesn't need to have 'Custom' included, just 'Name'. I've only added this comment to the name here, but I think it makes sense to have the settings key as just `name` as well.
```suggestion
            <label class="control-label">{{ _('Name') }}</label>
```